<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdTagsWidget</class>
 <widget class="QWidget" name="IdTagsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkSaveWhen">
     <property name="text">
      <string>Save when and where last seen</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="chkFastClock">
     <property name="text">
      <string>Use Fast Clock Time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>18</number>
     </property>
     <column>
      <property name="text">
       <string>System Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tag ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comment</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Where seen</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>When seen</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Roster Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
